config/clock.rb

```diff
@@ -25,6 +25,9 @@ class Clock
   every(1.hour, 'UpdateDuplicateMatchesWorker', at: '**:25') { UpdateDuplicateMatchesWorker.perform_async('notify_slack_at' => 13) }
   every(1.hour, 'DetectInvariantsHourlyCheck', at: '**:30') { DetectInvariantsHourlyCheck.perform_async }
 
+  # Warm the Vendor API cache for ApplicationChoices
+  every(23.hours, 'WarmCacheWorker') { WarmCacheWorker.perform_async }
```

Comment:
I'm having a hard time explaining my thinking here. 
Assuming that no requests are ever made and nothing other than this is writing to the cache, it would behave like this. 
| time       |                                                 |
|------------|-------------------------------------------------|
| 0          | no cache                                        |
| **23hrs**  | cache records created                           |
| **46hrs**  | cache from 23hrs ago is still valid - no change |
| 47hrs      | cached records expire (24hrs after created)     |
| **69hrs**  | cache records created                           |
| **92hrs**  | cache from 23hrs ago is still valid - no change |
| 93hrs      | cached records expire (24hrs after created)     |
| **115hrs** | cache records created                           |

Bold times are when this job runs. 

I think if we run this on a 24hr cycle it would work like
| time      |                                             |
|-----------|---------------------------------------------|
| 0         | no cache                                    |
| **24hrs** | cache records                               |
| 48hrs     | cached records expire (24hrs after created) |
| **48hrs** | cache records                               |
| 72hrs     | cached records expire (24hrs after created) |
| **72hrs** | cache records                               |

Reply:
added `*` to the times when the job is running

Reply:
If we run a job every 23 hours that just writes the cache (instead of fetching) we should be ok.